cypress/integration/functional/balances/balances.spec.ts

```diff
@@ -2,7 +2,7 @@ context('wallet/balances', () => {
   beforeEach(function () {
     cy.createEmptyWallet(true)
     cy.getByTestID('bottom_tab_settings').click()
-    cy.sendDFItoWallet().sendTokenToWallet(['BTC', 'ETH']).wait(4000)
+    cy.sendDFItoWallet().sendTokenToWallet(['DFI', 'BTC', 'ETH']).wait(10000)
```

Comment:
added new add DFI row in playground, it works just like adding any other token. However, the wait time has to be at least 6-8 sec to allow 2 block mining + some processing time.